h2o-bindings/bin/custom/python/gen_modelselection.py

```diff
@@ -225,3 +260,141 @@ def get_best_model_predictors(self):
 mode=maxr, the model returned is no longer guaranteed to have the best R2 value.
 """
 )
+
+examples = dict(
+    build_glm_model="""
+>>> import h2o
+>>> from h2o.estimators import H2OModelSelectionEstimator
+>>> h2o.init()
+>>> prostate = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/logreg/prostate.csv")
+>>> predictors = ["AGE", "RACE", "CAPSULE", "DCAPS", "PSA", "VOL", "DPROS"]
+>>> response = "GLEASON"
+>>> maxrModel = H2OModelSelectionEstimator(max_predictor_number=7,
+...                                        seed=12345,
+...                                        mode="maxr",
+...                                        build_glm_model=False)
+>>> maxrModel.train(x=predictors, y=response, training_frame=prostate)
+>>> results = maxrModel.result()
+>>> print(results)
+>>> coeff = maxrModel.coef()
+>>> print(coeff)
```

Comment:
replace line 266 to line 280 with:

>>> import h2o
>>> from h2o.estimators import H2OModelSelectionEstimator
>>> h2o.init()
>>> prostate = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/logreg/prostate.csv")
>>> predictors = ["AGE", "RACE", "CAPSULE", "DCAPS", "PSA", "VOL", "DPROS"]
>>> response = "GLEASON"
>>> maxrModel = H2OModelSelectionEstimator(max_predictor_number=5,
...                                        seed=12345,
...                                        mode="maxrsweep",
...                                        build_glm_model=True)
>>> result = maxrModel.result()
>>> one_model = h2o.get_model(result[“model_id”][ind, 0]) # get the GLM model with best performance for a fixed predictor size
>>> predict = one_model.predict(prostate) 
>>>print(predict) # print a portion of the predict frame

Reply:
I get a value error on the `result = maxrModel.result()` line.

Reply:
@wendycwong  here is the output from my error. I also added in a `model.train` line in case that was missing?

```
>>> prostate = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/logreg/prostate.csv")
>>> predictors = ["AGE", "RACE", "CAPSULE", "DCAPS", "PSA", "VOL", "DPROS"]
>>> response = "GLEASON"
>>> maxrModel = H2OModelSelectionEstimator(max_predictor_number=5, seed=12345, mode="maxrsweep", build_glm_model=True)
>>> maxrModel.train(x=predictors, y=response, training_frame=prostate)
>>> result = maxrModel.result()
>>> one_model = h2o.get_model(result[“model_id”][ind, 0])
  File "<stdin>", line 1
    one_model = h2o.get_model(result[“model_id”][ind, 0])
                                              ^
SyntaxError: invalid character in identifier
```